pandas/core/arrays/masked.py

```diff
@@ -22,12 +26,17 @@ class BaseMaskedArray(ExtensionArray, ExtensionOpsMixin):
     numpy based
     """
 
-    _data: np.ndarray
-    _mask: np.ndarray
-
     # The value used to fill '_data' to avoid upcasting
     _internal_fill_value: "Scalar"
 
+    def __init__(self, values: np.ndarray, mask: np.ndarray, copy: bool = False):
```

Comment:
\_\_init__ needs to be declared in the base class...

pandas\core\arrays\masked.py:56: error: Too many arguments for "BaseMaskedArray"
pandas\core\arrays\masked.py:181: error: Too many arguments for "BaseMaskedArray"
pandas\core\arrays\masked.py:207: error: Too many arguments for "BaseMaskedArray"
pandas\core\arrays\masked.py:207: error: Unexpected keyword argument "copy" for "BaseMaskedArray"
pandas\core\arrays\masked.py:213: error: Too many arguments for "BaseMaskedArray"
pandas\core\arrays\masked.py:213: error: Unexpected keyword argument "copy" for "BaseMaskedArray"

also creating this ensures that the subclasses have the correct signature for the constructor to work with `__invert__`, ` _concat_same_type`, `take` and `copy` from the base class.

we could just use a AbstractMethodError but I think it makes sense to put the shared functionality here.

BooleanArray has checking for values.ndim and mask.ndim. IntegerArray does not. It may make sense to have that check here also if applicable to IntegerArray.